python/rpdk/typescript/codegen.py

```diff
@@ -161,13 +161,23 @@ def generate(self, project):
 
         models = resolve_models(project.schema)
 
+        if project.configuration_schema:
+            configuration_models = resolve_models(
+                project.configuration_schema, "TypeConfigurationModel"
+            )
```

Comment:
would be great to achieve parity with other plugins:

```python
configuration_schema_path =project.root / project.configuration_schema_filename
project.write_configuration_schema(configuration_schema_path)
```

Reply:
@ammokhov Not sure what this does. As far as I know, we want all the models, `TypeConfiguration` included to be generated within `models.ts`. Am I missunderstanding something?